entgql/internal/todopulid/ent/schema/todo.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019-present Facebook
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	"fmt"
+
+	"github.com/facebook/ent"
+	"github.com/facebook/ent/schema/field"
+	"github.com/facebookincubator/ent-contrib/entgql/internal/todo/ent/schema"
+	"github.com/facebookincubator/ent-contrib/entgql/internal/todopulid/ent/schema/pulidgql"
+)
+
+// Todo defines the todo type schema.
+type Todo struct {
+	ent.Schema
+}
+
+// Mixin returns todo mixed-in schema.
+func (Todo) Mixin() []ent.Mixin {
+	return []ent.Mixin{
+		schema.Todo{},
+	}
+}
+
+func newPULIDFn(prefix string) func() string {
+	return func() string {
+		return fmt.Sprint(pulidgql.New(prefix))
+	}
+}
+
+// Fields returns todo fields.
+func (Todo) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("id").
+			DefaultFunc(newPULIDFn("TO")),
```

Comment:
Maybe use `GoType` here?  
```suggestion
		field.String("id").
			GoType(pulidgql. ID("").
			DefaultFunc(newPULIDFn("TO")),
```